R/SSplotComparisons.R

```diff
@@ -1415,7 +1420,8 @@ SSplotComparisons <-
     }
 
     if(!add){
-      axis(1, at=yr)
+      xticks <- pretty(xlim)
```

Comment:
The intent of x-axis ticks at the year values was that for surveys which don't occur annually to label the actual years where the observations occurred rather than a pretty set of more evenly spaced years. However, I now believe the best practice, as used for the hake assessment, is to include dummy observations (with negative fleet) in the intervening non-survey years to provide a smoother time series of expected survey values which eliminates this effect. So thanks for this change @kellijohnson-NOAA.